workinggroups/_posts/2015-11-04-training.md

```diff
@@ -3,20 +3,28 @@ title: Training
 layout: plain
 ---
 
+The HSF Training & Tutoring initiative is aimed at helping the research community (in various disciplines) to bridge the gap in computing skills existing nowadays between University level courses and what is required by scientist to produce high quality, sustainable software. 
 
+## WikiToLearn - collaborative textbooks
 
-## HSF and WikiToLearn
+To this extent we have adopted an already existing and successful platform, named [WikiToLearn](http://en.wikitolearn.org/Main_Page), originally developed in Italy for University students to deploy our tutoring in software but now growing in numbers also outside Italy. The basic principle of this initiative is that researchers can seldom afford to spend time writing down training material or professors transcribing their courses for the benefit of students in general. On the other hand students already take note during lessons or during their own academic research: it is therefore much easier for them to publish these notes and reports in a publicly visible site. WikiToLearn is a wikimedia-based content management system, pretty much like Wikipedia, were anybody can add new material or complete already existing one. To strengthen the quality of published material, scientists and researches can furthermore sign a particular version to enhance it's quality level. The platform is currently officially backed by Wikimedia end the KDE organization, and has received support from the University of Milano-Bicocca and the University of Pisa.
+Under the category 'Physics/Software' WikiToLearn now has a [branch](http://it.wikitolearn.org/Main_HSF_Page) dedicated to the HSF effort. We plan, in the coming months, to recruit as many volunteers as possible to publish there training material in various software domains and help the community to grow accordingly.
 
```

Comment:
I think this paragraph deserves some rewriting to make reading it easier... I'll try to come up with a modified version but would be even better done by a native english speaker (that I am not at all!).